lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
Shouldn't this only run on `.html` files? If the file begins its life as `md`, we'd only want to generate @mentions once it's HTML (to deal with things like code blocks). Is that possible?